projects/burnr/src/themes/substrate/light.js

```diff
@@ -1,184 +0,0 @@
-import { createMuiTheme }  from '@material-ui/core/styles'
```

Comment:
I think you can also delete `projects/burnr/src/themes/substrate/dark.js`. It's not in use anymore, right?

Reply:
done